Gemfile

```diff
@@ -23,5 +23,4 @@ group :test do
   gem 'activerecord'
   gem 'actionpack' # action_controller, action_view
   gem 'sprockets'
-  gem 'rspec-given'
```

Comment:
For some reason removing this also removed a warning that kept being thrown when `rspec` was ran. Might want to check to be sure the specs still run locally for y ou